### PR TITLE
Update package names of INBO package

### PR DESCRIPTION
**Pull request by ElsLommelen**
This PR makes some minor changes following changes in INBO packages (partly related to issue #37):
- add description of package `forrescalc`
- add description of package `inbodb`
- change package name for database related functions from `inborutils` to `inbodb`

(description of `inbodb` added to `retrieve data: biological data`)

### CODE REVIEW

File: content/articles/inbo_software/index.Rmd [hunk 1]
```diff
@@ -71,6 +72,10 @@ It also allows to download the data in a format that can easily be uploaded to [
 provides functionality to access and process data from the [European Tracking Network (ETN)](http://www.lifewatch.be/etn/) database hosted by the Flanders Marine Institute (VLIZ) as part of the Flemish contribution to LifeWatch.
 - **R package [n2khab](https://inbo.github.io/n2khab)**:
 provides preprocessed reference data _(including checklists, spatial habitat distribution, administrative & environmental layers, GRTSmaster_habitats)_ and preprocessing functions, supporting reproducible and transparent analyses on Flemish Natura 2000 (_n2k_) habitats (_hab_) and regionally important biotopes (RIBs).
+- **R package [forrescalc](https://github.com/inbo/forrescalc)**:
+provides aggregated values on dendrometry, regeneration and vegetation of the flemish forest reserve monitoring network, and functions to derive these data starting from individual tree measurements in Fieldmap.
+- **R package [inbodb]((https://inbo.github.io/inbodb)**:
+connect to and retrieve data from databases on the INBO server, with dedicated functions to query databases INBOVEG and Florabank.
```
[the remark this answers is not in the document]
Comment (ElsLommelen) [reply content/articles/inbo_software/index.Rmd:78]: That was also my first consideration, but as newly added databases may contain environmental data, it is likely that the whole description has to be moved anyway after adding new databases...  So I rather wait for the decision on the best category for this package.  (Strange: I added this to our discussion on issue #37 yesterday, but is has disappeared.)

[the remark this answers is not in the document]
Comment (florisvdh) [reply content/articles/inbo_software/index.Rmd:78]: > Strange: I added this to our discussion on issue #37 yesterday, but is has disappeared

Never seen that :thinking:, probably didn't come through.

[the remark this answers is not in the document]
Comment (ElsLommelen) [reply content/articles/inbo_software/index.Rmd:78]: I wrote it shortly before the github downtime yesterday, I'll redo it.

File: content/articles/inbo_software/index.md [hunk 2]
```diff
@@ -30,19 +30,19 @@ the package’s purpose and demonstrating its use.
 
 The following table gives a **quick overview**:
```
Comment (florisvdh) [inline content/articles/inbo_software/index.md:31]: I suggest to delete in the md file the many generated extra spaces (cf. my earlier comment: use the one from the Rmd file)

File: content/articles/inbo_software/index.md [hunk 3]
```diff
@@ -52,6 +52,13 @@ The following table gives a **quick overview**:
 
 ## Retrieve data
 
+### General data
```
Comment (florisvdh) [inline content/articles/inbo_software/index.md:55]:
```suggestion
### General
```

**Comment by florisvdh** [comment]
Thanks a lot @ElsLommelen, feel free to merge if you are ready to.

**Pull request merged**